unlock_assets.sh

```
@@ -1,11 +1,22 @@
+  echo "Required config.sh or lib.sh not found" >&2
+  exit 1
+fi
+
+source "$CONFIG_FILE"
+source "$LIB_FILE"
 
-SELECT_UTXO="./select-utxo.sh"
 FILE_UTILS="./file_utils.sh"
 
 # Import modules
 source "$FILE_UTILS"
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)